.travis.yml

```diff
@@ -2,10 +2,12 @@ language: node_js
 
 sudo: required
 
+services:
```

Comment:
Travis CI was failing because of this update https://changelog.travis-ci.com/ubuntu-xenial-16-04-build-environment-update-86123. I applied this change according to the docs.